WeakAurasOptions/AuthorOptions.lua

```diff
@@ -1420,7 +1420,7 @@ local function ensureNonDuplicateKey(option)
   end
 end
 
-function addAuthorModeOption(options, args, data, order, prefix, i)
+function addAuthorModeOption(options, args, data, order, prefix, i, isSubOption)
```

Comment:
If we do decide we want this then instead of having a flag, have an integer which represents the depth of recursion. A depth > 0 indicates that this is some sub option, otherwise it is not.

Reply:
Also unless I’m mistaken the value is never set anywhere so you may as well remove it.